container-image/examples/podman/README.md

```diff
@@ -0,0 +1,40 @@
+#  Running the image locally using podman and the host network
+
+* Starts the WildFly server you want to interact with. Management interface should be secured with a user and password.
+
+* Starts the 'ollama' container with `qwen2.5:3b` loaded.
+
+```shell
+podman run -d --rm --name ollama --replace --pull=always -p 11434:11434 -v ollama:/root/.ollama --stop-signal=SIGKILL mirror.gcr.io/ollama/ollama
+
+podman exec -it ollama ollama run qwen2.5:3b
+```
+To quit the Ollama prompt, type **/bye**.
+```
+
+* Starts the image (replace the user name and password with yours):
+
+```
+podman run -e OLLAMA_CHAT_URL=http://localhost:11434 \
+--network=host -e PORT_OFFSET=10 \
+-e WILDFLY_MCP_SERVER_USER_NAME=<your admin user> -e WILDFLY_MCP_SERVER_USER_PASSWORD=<your admin password> \
+quay.io/jdenise/wildfly-chat-bot:latest
```

Comment:
should the image be `wildfly/wildfly-chat-bot` instead (based on https://github.com/wildfly-extras/wildfly-mcp/pull/12/files#diff-eb60d4cc52b387b0c6ea3a1dc96add370a6190389e83bcd68aa1ac020a91b90bR16)